pkg/ceph/rbd/connection.go

```diff
@@ -44,7 +42,3 @@ func NewConnection(monitor, id, key, pool, datapool string) (*Connection, error)
 		DataPool: datapool,
 	}, nil
 }
-
-func (c *Connection) Destroy() error {
-	return os.Remove(c.KeyFile)
```

Comment:
I meant only for the directory. you still need to cleanup keyfile or else its a security issue